src/main.c

```diff
@@ -466,18 +470,22 @@ void LCUI_InitApp(LCUI_AppDriver app)
 	LCUICond_Init(&MainApp.loop_changed);
 	LCUIMutex_Init(&MainApp.loop_mutex);
 	LinkedList_Init(&MainApp.loops);
+	LCUIProfile_Init(&MainApp.profile);
+	MainApp.settings_change_handler_id = LCUI_BindEvent(
+	    LCUI_SETTINGS_CHANGE, OnSettingsChangeEvent, NULL, NULL);
+	Settings_Init(&MainApp.settings);
```

Comment:
This problem is still not solved, I suggest inserting `printf("fps: %d, fps_limit: %d\n", LCUI_GetFrameCount(), MainApp.settings.frame_rate_cap)` in the main loop to confirm the cause of the problem.

![image](https://user-images.githubusercontent.com/1730073/86219520-e80e9a80-bbb4-11ea-822d-bda2b61a04fd.png)